dita/regions/europe/spain/unit_db.dita

```diff
@@ -0,0 +1,156 @@
+                            <entry namest="col3" nameend="col4">H1-4</entry>
+                            <entry namest="col5" nameend="col6">Discrete. Doublets
+                                common</entry>
+                        </row>
+                        <row>
+                            <entry>BR</entry>
+                            <entry>S5</entry>
+                            <entry namest="col3" nameend="col4">H1</entry>
+                            <entry namest="col5" nameend="col6">Discrete. Doublets
+                                common</entry>
+                        </row>
+                        <row>
+                            <entry>Cat b Engine R</entry>
+                            <entry>2-1000.4</entry>
+                            <entry namest="col3" nameend="col4">H2, 4, 21, 145</entry>
+                            <entry namest="col5" nameend="col6">The quick brown fox does
+                                something similar</entry>
+                        </row>
+                        <row>
+                            <entry>Unk BBA</entry>
+                            <entry>25 &amp; CSR</entry>
+                            <entry namest="col3" nameend="col4">H1</entry>
+                            <entry namest="col5" nameend="col6">CBR 34</entry>
+                        </row>
+                        <row>
+                            <entry>Unk BBA</entry>
+                            <entry>135 &amp; CSR</entry>
+                            <entry namest="col3" nameend="col4">H1 &amp; 2</entry>
+                            <entry namest="col5" nameend="col6">CBR 54334</entry>
+                        </row>
+                        <row>
+                            <entry namest="col1" nameend="col6">Signature &amp;
+                                Characteristics</entry>
+                        </row>
+                        <row>
+                            <entry>Acoustic Signature</entry>
+                            <entry namest="col2" nameend="col6">Expect plant to be similar
+                                to 2016 steam powered version</entry>
+                        </row>
+                        <row>
+                            <entry>Unique classifiers</entry>
+                            <entry namest="col2" nameend="col6">The dominant frequency will
+                                be Laboris nostrud ipsum ex tempor. Est fugiat et esse
+                                excepteur est voluptate cupidatat in sit. Ex est laborum
+                                commodo eiusmod. Et tempor labore dolor eu laborum labore ex
+                                sit deserunt cillum sit duis minim. Nisi ea ullamco minim
+                                duis elit consectetur mollit ea mollit sunt voluptate. Esse
+                                nulla id mollit ad deserunt Lorem deserunt excepteur
+                                nostrud. <image href="images/Image.jpg"/> Figure 2.
```

Comment:
could you use one of the images from the mock data please?

Reply:
updated